ext/curl/curl.stub.php

```diff
@@ -26,6 +26,27 @@ final class CurlShareHandle
 {
 }
 
+#if LIBCURL_VERSION_NUM >= 0x073E00 /* Available since 7.62.0 */
+/**
+ * @strict-properties
+ * @not-serializable
+ */
+final class CurlUrl implements Stringable
+{
```

Comment:
I'm wondering whether we want to declare properties, just like what a normal class would do.